<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SynthereDub</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startbut">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="select_inputpath">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>13</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_savepath">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>15</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SaveTo</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputpath_2">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>23</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Your selected video </string>
    </property>
   </widget>
   <widget class="QLabel" name="savepath">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>25</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Save the translated video and txt to</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Original language</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>101</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Translate to</string>
    </property>
   </widget>
   <widget class="QComboBox" name="origin_lang">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>EN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AutoDetect</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="target_lang">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CN</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>102</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Role1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="role1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>zh-CN-YunxiNeural</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zh-CN-YunjianNeural</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zh-CN-XiaoxiaoNeural</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zh-CN-XiaoyiNeural</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="preview_role">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>99</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="translate_process_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>91</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translator</string>
    </property>
   </widget>
   <widget class="QComboBox" name="translator">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>microsoft</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radio_use_tts">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Use EDGE TTS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_use_audio">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_audio_file_but">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Audio</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>54</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Role2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="role2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>zh-CN-YunjianNeural</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zh-CN-YunxiNeural</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zh-CN-XiaoxiaoNeural</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zh-CN-XiaoyiNeural</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Speaker Num</string>
     </property>
    </widget>
    <widget class="QComboBox" name="speaknum_box">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="ref_audio_file_lab">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected audio file</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>startbut</zorder>
   <zorder>select_inputpath</zorder>
   <zorder>select_savepath</zorder>
   <zorder>inputpath_2</zorder>
   <zorder>savepath</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>origin_lang</zorder>
   <zorder>target_lang</zorder>
   <zorder>label_3</zorder>
   <zorder>role1</zorder>
   <zorder>preview_role</zorder>
   <zorder>translate_process_bar</zorder>
   <zorder>label_4</zorder>
   <zorder>translator</zorder>
   <zorder>radio_use_tts</zorder>
   <zorder>radio_use_audio</zorder>
   <zorder>select_audio_file_but</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
